src/module.ts

```diff
@@ -78,17 +77,3 @@ export default defineNuxtModule<ModuleOptions>({
     })
   }
 })
-
-declare module '@nuxt/schema' {
```

Comment:
@pi0 Is `@nuxt/module-builder` taking care of `publicRuntimeConfig`, is it safe to remove?

Reply:
Not at the moment but totally open to introducing shortcut support like for type exports.